tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t3.py

```diff
@@ -0,0 +1,620 @@
+            condition_function=validators.oneOf((1, 2)),
+            result_field="RACE_WHITE",
+            result_function=validators.isInLimits(1, 2),
+        ),
+        validators.if_then_validator(
+            condition_field="FAMILY_AFFILIATION",
+            condition_function=validators.oneOf((1, 2)),
+            result_field="RELATIONSHIP_HOH",
+            result_function=validators.isInStringRange(4, 9),
+        ),
+        validators.if_then_validator(
+            condition_field="FAMILY_AFFILIATION",
+            condition_function=validators.oneOf((1, 2)),
+            result_field="PARENT_MINOR_CHILD",
+            result_function=validators.oneOf((2, 3)),
+        ),
+        validators.if_then_validator(
+            condition_field="FAMILY_AFFILIATION",
+            condition_function=validators.matches(1),
+            result_field="EDUCATION_LEVEL",
+            result_function=validators.notMatches("99"),
+        ),
+        validators.if_then_validator(
+            condition_field="FAMILY_AFFILIATION",
+            condition_function=validators.matches(1),
+            result_field="CITIZENSHIP_STATUS",
+            result_function=validators.oneOf((1, 2)),
+        ),
+        validators.if_then_validator(
+            condition_field="FAMILY_AFFILIATION",
+            condition_function=validators.matches(2),
+            result_field="CITIZENSHIP_STATUS",
+            result_function=validators.oneOf((1, 2, 9)),
+        ),
+    ],
+    fields=[
+        Field(
+            item="0",
+            name="RecordType",
+            friendly_name="record type",
+            type="string",
+            startIndex=0,
+            endIndex=2,
+            required=True,
+            validators=[],
+        ),
+        Field(
+            item="4",
+            name="RPT_MONTH_YEAR",
+            friendly_name="reporting month and year",
+            type="number",
+            startIndex=2,
+            endIndex=8,
+            required=True,
+            validators=[],
+        ),
+        Field(
+            item="6",
+            name="CASE_NUMBER",
+            friendly_name="case number",
+            type="string",
+            startIndex=8,
+            endIndex=19,
+            required=True,
+            validators=[validators.isAlphaNumeric()],
+        ),
+        Field(
+            item="66",
+            name="FAMILY_AFFILIATION",
+            friendly_name="family affiliation",
+            type="number",
+            startIndex=19,
+            endIndex=20,
+            required=True,
+            validators=[validators.oneOf([1, 2, 4])],
+        ),
+        Field(
+            item="67",
+            name="DATE_OF_BIRTH",
+            friendly_name="date of birth",
+            type="number",
+            startIndex=20,
+            endIndex=28,
+            required=True,
+            validators=[
+                validators.dateYearIsLargerThan(1950),
```

Comment:
```suggestion
                validators.dateYearIsLargerThan(1900),
```

Reply:
#2746

Reply:
recommended 1900 just for consistency in the codebase across sections 1 and 2, the latter of which does not distinguish between children and adults.